user_service/user_service/settings.py

```diff
@@ -75,12 +79,15 @@
 
 DATABASES = {
     'default': {
-        'ENGINE': 'django.db.backends.sqlite3',
-        'NAME': BASE_DIR / 'db.sqlite3',
+        'ENGINE': 'django.db.backends.postgresql',
+        'NAME': 'admin',
+        'USER': 'postgres',  
+        'PASSWORD': 'postgres',  
```

Comment:
A bit trivial, but you might want to use .env variables for passwords

Reply:
Indeed, using a .env file will be a good practice. I have updated this PR accordingly.